src/lib/utils/driver.ts

```diff
@@ -0,0 +1,108 @@
+import { driver } from 'driver.js';
+
+import { ACTION_DOWNLOAD, ACTION_ENROLE, ACTION_IMPORT } from '$lib/Enum/actions';
+import { STATUS_OK, STATUS_PENDING, STATUS_UNEXPECTED, STATUS_UNREACHABLE } from '$lib/Enum/status';
+
+export const driverObj = driver({
```

Comment:
```suggestion
export const driverjs = driver({
```

Ou quelque chose comme ça